app/assets/javascripts/administrate/components/date_time_picker.js

```diff
@@ -1,5 +1,9 @@
 $(function () {
-  $(".datetimepicker").datetimepicker({
+  $('[data-type="datetime"]').datetimepicker({
+    debug: false,
+    format: "HH:mm:ss",
+  });
+  $('[data-type="time"]').datetimepicker({
```

Comment:
This looks like it's in the wrong order. Shouldn't this be `datetime` and the above be `time`?